[ServiceBus] prep release 7.12.0 (#34543)
Azure/azure-sdk-for-python: sdk/servicebus/azure-servicebus/CHANGELOG.md

# Release History

## 7.12.0 (2023-02-06)

### Features Added

> - Updated `max_wait_time` on the ServiceBusReceiver constructor to additionally configure the server timeout for Session-Enabled/Queues/Topics when NEXT_AVAILABLE_SESSION is set, allowing users to set a different timeout than the default 65 seconds when accepting a session.

Reviewer: ```suggestion
- Updated `max_wait_time` on the ServiceBusReceiver constructor allowing users to change the default server timeout of 65 seconds when accepting a session Session-Enabled/Queues/Topics when NEXT_AVAILABLE_SESSION is used,
```
